packages/amplify_core/lib/src/types/auth/mfa/user_mfa_preference.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_core/src/config/auth/cognito/mfa.dart';
+import 'package:aws_common/aws_common.dart';
+
+/// {@template amplify_core.auth.user_mfa_preference}
+/// The MFA preferences for the user.
+/// {@endtemplate}
+final class UserMfaPreference
+    with
+        AWSSerializable<Map<String, Object?>>,
+        AWSEquatable<UserMfaPreference>,
+        AWSDebuggable {
+  /// {@macro amplify_core.auth.user_mfa_preference}
+  const UserMfaPreference({
+    Set<MfaType>? enabled,
+    this.preferred,
+  }) : enabled = enabled ?? const {};
```

Comment:
I was wanting passing `null` before because it makes things like `list?.map(...).toSet()` cleaner if I don't have to duplicate `?? const {}` everywhere. But I changed it since I'm not doing that any more.